README.md

```diff
@@ -0,0 +1,22 @@
+# mygnuboy
+My own version of gnuboy emulator
+
+
+WHEN COMPILING AND INSTALLING:
+
+Remove and insert the following flags in the makefile that gets generated by ./configure:
+
+	NOTICE: all the dashes are part of the commands
+
+	-fforce-mem  -----> must be removed if compilation is not successful
+	-std=c99 -----> necessary because there are a few comments starting with //
+        -pthread -----> I have implemented pthreads, so it is necessary too
+
+
+NEW DEPENDENCIES ADDED:
+
+
+	scrot
+	ffmpeg
```

Comment:
if you want to add third-party dependency, then please make them optional. ffmpeg is a huge download and takes considerable time to build.